manifests/inet_filter.pp

```diff
@@ -1,7 +1,19 @@
 # manage basic chains in table inet filter
 class nftables::inet_filter inherits nftables {
 
-  $_log_prefix_discard = sprintf($nftables::log_prefix, { 'chain' => '%<chain>s', 'comment' => 'Rejected: ' })
+  $_reject_rule_template = @(END)
+<% if $log_limit { -%>
+limit rate <%= $log_limit %> log prefix "<%= $log_prefix %>" flags all counter
+<% } else { -%>
+log prefix "<%= $log_prefix %>" flags all counter
+<% } -%>
+END
+  $_reject_rule = inline_epp($_reject_rule_template,
```

Comment:
can we move that to an external template, since this really clutters the readability of the class

Reply:
No problem, doing :)